job_manager/job_manager.py

```diff
@@ -1,5 +1,5 @@
 from redis import Redis
-from rq import Queue
+from rq import Queue, Retry
```

Comment:
whats retry?

Reply:
In the 'enqueue' method below, the Retry object lets RQ schedule a task to run again if it fails. If you look in that line, there are three numbers indicating how many seconds RQ will wait from the time of failure to try again. There are three opportunities, if the job fails a fourth time it doesn't get another chance. These retrys are needed because for large sets of workers all doing queries at the same time, Augur will close the connection on some of them because it's reached its query limit.